### DISCUSSION TITLE
GFR fraction and renal impairment and enzyme abundance

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1333] [Questions & Problems]
Hello, everyone. I recently modeled a study involving people with renal insufficiency, and did not find out if GFR Fraction changes when renal insufficiency occurs. I would like to ask you about GFR fraction and the changes in the abundance of CYP2B6, UGT2B7 and CYP2C8 enzymes in response to renal impairment.

**Comment by prvmalik** [comment]
Interesting question. I think GFR fraction could stay the same, although the maximal filtration rate will be modulated through the increase in fraction unbound.

CYP2C8 changes are covered by papers by Tan et al., 2018, PMID: 28990182. You could start there.

For isozymes without convincing data, you could use the averages determined by Sayama et al., 2014, PMID: 24912798.

As always, perform a sensitivity analysis across different assumptions for these parameters to quantify the uncertainty in your conclusions that is attributed to the degree of your uncertainty in these parameterizations.

Paul

**Reply by WTY-tt** [reply]
Thank you for your reply. I will further study the article you mentioned.
